nltk/test/corpus_fixt.py

```diff
@@ -1,3 +0,0 @@
-# -*- coding: utf-8 -*-
-
-from nltk.corpus import teardown_module
```

Comment:
i was confused by files like these. they imported the teardown, but that teardown wasn't used in the fixture or corresponding doctest file